src/main/java/com/staffbase/plugins/sdk/sso/SSOData.java

```diff
@@ -439,10 +449,20 @@ public boolean isDeleteInstanceCall() {
     return Optional.ofNullable(this.tags);
   }
 
+  public String getBranchId() {
+    return this.branchId;
+  }
+
+  public String getBranchSlug() {
+    return this.branchSlug;
+  }
+
   @Override
   public String toString() {
     return "SSOData ["+
   " instanceID="+ this.instanceID+
+  ", branchId="+ this.branchId+
```

Comment:
The SDK currently uses ID instead of Id. For consistency reasons, I'd recommend to use `branchID`.

Switching everything to `Id` should be a separate PR.